lib/lambda_ethereum_consensus/application.ex

```diff
@@ -29,6 +29,12 @@ defmodule LambdaEthereumConsensus.Application do
       {LambdaEthereumConsensus.Libp2pPort, libp2p_opts},
       {LambdaEthereumConsensus.Store.Db, []},
       {LambdaEthereumConsensus.P2P.Peerbook, []},
+      {LambdaEthereumConsensus.ForkChoice, [checkpoint_sync]},
+      {LambdaEthereumConsensus.P2P.IncomingRequests, []},
+      {LambdaEthereumConsensus.Beacon.PendingBlocks, []},
+      {LambdaEthereumConsensus.Beacon.SyncBlocks, []},
+      {LambdaEthereumConsensus.P2P.GossipSub, []},
+      {LambdaEthereumConsensus.P2P.Metadata, []},
```

Comment:
I think there are some merge conflicts here 🤔. Recently we moved some of these to `LambdaEthereumConsensus.Beacon.BeaconNode`

Reply:
hey I don't know how that happened actually just fixed it

Reply:
It's just something `git` does sometimes 😞